laterpay/application/Controller/Post.php

```diff
@@ -1058,6 +1062,10 @@ public function modify_post_content( $content ) {
 
         // add the teaser content
         $html .= LaterPay_Helper_View::remove_extra_spaces( $this->get_text_view( 'frontend/partials/post/teaser' ) );
+        
+        if ( $only_time_passes_allowed ) {
+            $html .= LaterPay_Helper_View::remove_extra_spaces( $this->get_text_view( 'frontend/partials/post/after_teaser' ) );
```

Comment:
Oh, sorry, my bad. Added in https://github.com/laterpay/laterpay-wordpress-plugin/commit/812540bd16ab46510a950d784f4727e0f9fd1625